src/test/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheckTest.java

```diff
@@ -166,29 +164,29 @@ public void testLogOutput() throws Exception {
         final String secondErrorMessage = getCheckMessage(MSG_KEY, "anotherKey");
 
         verify(checker, propertyFiles, ImmutableMap.of(
-            getPath(""), Collections.singletonList(line + firstErrorMessage),
-            translationProps, Collections.singletonList(line + secondErrorMessage)));
-
-        final String osName = System.getProperty("os.name").toLowerCase(ENGLISH);
-        String expectedLogOutput;
-
-        // till https://github.com/checkstyle/checkstyle/issues/5103
-        if (osName.startsWith("windows")) {
-            expectedLogOutput = readFile("OutputTranslationCheckWindows.xml");
-        }
-        else if (osName.startsWith("linux")) {
-            expectedLogOutput = readFile("OutputTranslationCheckLinux.xml");
-        }
-        else {
-            expectedLogOutput = readFile("OutputTranslationCheckMacOS.xml");
-        }
-
-        expectedLogOutput = expectedLogOutput.replace("path_to_file", getPackageAbsolutePath());
-        expectedLogOutput = String.format(Locale.getDefault(), expectedLogOutput,
-                firstErrorMessage, secondErrorMessage).replace("'", "&apos;");
-
-        assertEquals("Unexpected log output", expectedLogOutput,
-            out.toString(StandardCharsets.UTF_8.name()));
+            ":0", Collections.singletonList(" " + firstErrorMessage),
```

Comment:
@romani Please examine this line and help me understand why I had to change it so much.
Original Code:
````
getPath(""), Collections.singletonList(line + firstErrorMessage), // line = "0: ";
````
Expected replacement code:
````
"", Collections.singletonList(line + firstErrorMessage),
````
Actual replacement code:
````
":0", Collections.singletonList(" " + firstErrorMessage),
````

I was required to make file name be ":0" and add a space before `firstErrorMessage` otherwise the test would fail.
I haven't looked into this deeply, but it seems a weird change but I assume it is required because I added `checker.setBasedir(getPath(""));`.
But why ":0"?

Reply:
I think I understand once I printed out the display.
````
:0: Properties file 'InputTranslationCheckFireErrors_ja.properties' is missing.
InputTranslationCheckFireErrors_de.properties:0: Key 'anotherKey' missing.
Audit done.
````

https://github.com/checkstyle/checkstyle/blob/3c1ddf428d9ddecaff63986007b4f5ff0593c098/src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java#L352-L354
It doesn't realize that `:0` is the line number and incorrectly assumes it is part of the file name because we have to support windows and it's drive letters like `C:\`.
So that explains it.